sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -478,10 +478,14 @@ def profiles_sample_rate=(profiles_sample_rate)
       @profiles_sample_rate = profiles_sample_rate
     end
 
+    def event_building_allowed?
```

Comment:
@st0012 lmk if you have a better name for this :)

Reply:
I feel this should be called `sending_allowed?` and the original `sending_allowed?` should be something like `sending_to_sentry?`